packages/esify/lib/codemod.js

```diff
@@ -52,6 +53,7 @@ var TRANSFORMS = [
   {path: 'shopify-codemod/transforms/computed-literal-keys-to-dot-notation'},
   {path: 'shopify-codemod/transforms/rename-identifier'},
   {path: 'shopify-codemod/transforms/rename-property'},
+  {path: 'shopify-codemod/transforms/split-return-assignments'},
```

Comment:
A transform from moons ago.  It was either never added to this list, or was lost in a bad merge.  Either way, it didn't break anything when I added it back, so I left it in here.